tests/test_idutils.py

```diff
@@ -51,6 +51,12 @@
     ('doi.org/10.1016/j.epsl.2011.11.037', ['doi', 'handle'],
         '10.1016/j.epsl.2011.11.037',
         'https://doi.org/10.1016/j.epsl.2011.11.037'),
+    ('10.1016/üникóδé-дôΐ', ['doi', 'handle'],
+        '10.1016/üникóδé-дôΐ',
+        'https://doi.org/10.1016/üникóδé-дôΐ'),
```

Comment:
As far as I understand [DOI character encoding specification](https://www.doi.org/doi_handbook/2_Numbering.html#htmlencoding) we would need to URL-encode the non-ASCII characters.